clang/lib/Headers/ia32intrin.h

```diff
@@ -173,25 +183,59 @@ __popcntq(unsigned long long __A)
 #endif /* __x86_64__ */
 
 #ifdef __x86_64__
+/// Returns the program status and control \c RFLAGS register with the \c VM
+///    and \c RF flags cleared.
+///
+/// \headerfile <x86intrin.h>
+///
+/// This intrinsic corresponds to the \c PUSHFQ + \c POPQ instruction sequence.
```

Comment:
Also `POP` instead `POPQ`. The same below.

Reply:
Done